nextjs-13/src/app/send/page.tsx

```diff
@@ -0,0 +1,116 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import { ClipLoader } from 'react-spinners';
+
+import MultiSelectWrapper from '@components/MultiSelectWrapper';
+import requestManager from 'lib/request/manager';
+import makeToast from 'lib/toast/makeToast';
+
+const SendNewsletter = () => {
+  const [selected, setSelected] = useState([]);
+  const [records, setRecords] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [email, setEmail] = useState('');
+
+  const getNewletters = async () => {
+    setIsLoading(true);
+
+    try {
+      const response = await requestManager('GET', 'v1/newsletter');
+
+      setIsLoading(false);
+      setRecords(response.records);
+    } catch (error) {
+      makeToast(error.message, 'error');
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    getNewletters();
+  }, []);
+
+  const afterSubmit = () => {
+    setIsLoading(false);
+    setSelected([]);
+    setEmail('');
+    getNewletters();
+  };
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    if (!selected.length) {
+      return makeToast('Please select atleast one newsletter', 'error');
```

Comment:
Minor typo:

```suggestion
      return makeToast('Please select at least one newsletter', 'error');
```

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/123/files#diff-1410c2c61c8257ce65caa55dcf666a7b6315edc7ddd9c166d4b608a8fe552febR45) and [here](https://github.com/nimblehq/js-framework-benchmark/pull/123/files#diff-79f2f429dd86ce329745b8c7cd2baf8ffbaa9bc8541c610395d99a79eb0dfea5R109)